### DISCUSSION TITLE
OSC2024 東京秋（10/26 リアル展示）へのブース出展について

### DISCUSSION THREAD

**Discussion by potappo** [mozilla-japan/translation#786] [General]
前回 https://github.com/mozilla-japan/translation/discussions/737

OSC2024 東京秋の日程は 2024年10月26日(日) 10:00～16:00 です。ブース出展募集締め切りは9/24です。
ぎりぎり 9 月の MDN 翻訳ミートアップでも間に合いますが、今日のミートアップの中間報告会で調整することが特になければ、本日 8/24 か 8/25 には申込してしまう予定です。申込が受理されたら報告します。**【8/26 追記】8/24申込、8/26受付済です。**
https://event.ospn.jp/osc2024-fall/

場所は春と同じ、浅草駅最寄の東京都立産業貿易センター台東館です。

会場のルールは春と同じで、展示用にネットワークが使えます。ストーリミング動画再生は負荷になると思うので、前回の動画ファイルをPCに入れて持っていきます。MDN や GitHub へのアクセスは問題ないですが、使えるようになるまで少し時間かかるかもしれません（いつも会場内で使えるようになったかのアナウンスがあります）。

会場宛の宅配便は使えないので、展示物・配布物はすべて出展者が直接持ち込みになります。昼食は建物内に食堂があります。
出展を手伝っていただける方はこのディスカッションに連絡ください。時間調整のための調整さんを後日作成予定です。
※調整さん作成は9/21のミートアップ時で良いかもと考えています

スケジュールは以下です。前日準備はありません。ゴミは持ち帰りとなります。
当日準備：9:00-10:00
展示時間：10:00-16:00
撤収時間：16:00-17:00（17:00完全撤収）

机のサイズは以下となっています。机1台、椅子2脚で申し込みます。
机　：0.5〜1台（150センチx60センチ）
椅子：原則2脚（1名のブースは1脚）

**Comment by potappo** [comment]
コミュニティ紹介チラシに Youtube 動画 の URL を加えて新しいものにする予定です（まだ作業未着手）。

**Reply by potappo** [reply]
修正案
- 既存の URL をすべて QR コードにして並べてみる
- 文字の大きさを小さくして 1 行足す

来月のミートアップ (9/21) までに修正予定、それまでに時間が取れなかった場合は 9/21 に作業

**Reply by potappo** [reply]
9/21 午前中に修正し始めました。今日のミートアップで完成を目指します。

**Reply by potappo** [reply]
修正完了したチラシのPDFです。井上さんに 50 部印刷依頼しました。
https://drive.google.com/file/d/1942LtpaSY-IJMW9zZRP-q9uNQ72l3V9K/view?usp=sharing

**Reply by potappo** [reply]
井上さんにロゴを調整していただいたので、そちらをバックアップ兼ねて、下記に公開します。
https://drive.google.com/file/d/17d4_lpNyvDJCnQU2Q1Ot9hDM_jDWEvnt/view?usp=sharing

**Comment by potappo** [comment]
申込書を OSC 事務局に送りました。申込日付以外、春と同じ記載内容です。

**Reply by potappo** [reply]
申込受付されました。

以下が展示ブース一覧のページです。
https://event.ospn.jp/osc2024-fall/exhibit

**Comment by hmatrjp** [comment]
グレさんへの荷物発送は、10月第2週発送を目標にします。ずらしたほうが良ければお知らせください。　 @gurezo

**Reply by gurezo** [reply]
@hmatrjp 
ありがとうございます！
発送後に伝票番号を共有してもらえれば、後はこちらで調整します！！！

**Reply by hmatrjp** [reply]
追跡番号 7522-2112-9321 で発送しました。着予定は明後日の午後だそうです。

内容物は、
・ストラップ 20
・R シール 30
・Mozilla シール（黒地に白字） 30
・アルミスタンド 4
・マグカップ 2 （グレさんとpotappoさん用）
です。

丸いFirefox シール  （切り離す必要あり） は、発送時バタバタしていて入れ損ねました。 😫

**Reply by gurezo** [reply]
@hmatrjp 
手配有難うございます。

> 追跡番号 7522-2112-9321

こちら郵便の追跡番号であっていますか？（確認
ヤマト便も自分はよく使うので。

**Reply by gurezo** [reply]
@hmatrjp 

郵便局から連絡が来ていました！受け取り日時は調整します！
手配有難うございました！

**Reply by gurezo** [reply]
@hmatrjp cc @potappo 
今日無事に受け取りました！

**Reply by hmatrjp** [reply]
[ノベルティグッズ管理表](https://docs.google.com/spreadsheets/d/1N73mciqvlQbuC4457OMteS-TXDldj75XybDKfK8CToI/edit?gid=345022493#gid=345022493)のR列に、今回の出庫数を入れました。

**Comment by potappo** [comment]
今のところ、ブース展示参加者は私とグレさんだけなので、調整さん立てなくても良いかも（当日の来場者具合で調整）と考えています。
調整さんがあった方が良い、追加で展示手伝い希望の方がいればこちらでお知らせください。

**Reply by gurezo** [reply]
@potappo 
はい。それで良いと思います。
ブース番が増える分には、問題ないと思います！

**Comment by potappo** [comment]
テーブルクロスですが、既に納品されているとのことなので、留め金（クロス用クリップ）と一緒に私の自宅に送ってもらうということになりました。

テーブルクロスの他に、展示用 PC、チラシ、前回余ったグッズも持っていくので、揃い次第、カバンに入れてみて、会場への運搬体制は考えます。前回のようにキャリーバッグも使えば余裕ありと思いますが、大きめのビジネスバッグ 1 つで済むなら、ビジネスバッグ 1 つにするかもです。

なお、展示用 PC は MDN コミュニティにしか貸し出ししていない PC とのことなので、私預かりにすることになりました。
展示用 PC の行き来がなくなることで両者の手間が軽減される以外にメリットはありません。
動作確認兼ねて Windows Update を毎月するぐらいのメンテナンスはしますが、他は毎回の出展毎の更新になるかと思います。

**Reply by potappo** [reply]
今日手元に届きました。中身が揃っていることまでは確認しましたが、明日は朝から忙しいため、テーブルクロスの細かい確認は早くて金曜日、遅くて土曜日以降になります。

**Reply by potappo** [reply]
テーブルクロスは折りたたんだ状態でソフトファイルケースに入っていましたので、ケース入りのまま、カバンに入れて運搬する予定です。
広げるのに適したテーブルやスペースがないので、テーブルクロスを広げた写真は上げられませんが、同梱されていた留め金はこんな感じという写真を一応上げておきます。
![留め金](https://github.com/user-attachments/assets/7b5fbbd3-4b0d-4875-817f-3289f29ba77e)

**Reply by potappo** [reply]
すべてカバン一つに収まりましたので、当日はそれで行きます。

**Comment by potappo** [comment]
展示用PCをセットアップ中です。当日は動画だけ流す可能性もなくはないですが、前回までと同様に、MDN ローカルプレビュー環境や Web 版翻訳ガイドをローカルで見れるようにしておきます。

なんとなくそうかなとは思っていましたが、この PC は Windows 11 にはアップグレードできない（CPU が
 Intel 第7世代で対応 CPU ではない）ため、来年 10 月 14 日で OS 的には EOL となります。春出展時は使えますが、秋出展時には交換が必要とだけはメモしておきます。 春出展終了以降のいつ返却するかは来年の私の忙しさ次第になります。Windows Update だけするぐらいなら早々に返すか、余裕があれば、別地方の OSC に出展する可能性もあります。

**Reply by gurezo** [reply]
@potappo 
対応有難うございます！

地方出展は、落ち着いてからタイミングや地域含めて、みんなでそうだんしましょう！！

**Comment by potappo** [comment]
終了後タイミングを逃してしまい遅くなりましたが、今回の出展報告です。

チラシは 50 部用意して残数 15 枚ですので、35 枚配りました。ブースに来てくれた方も概ね同じ人数になるかと思います。

グッズ在庫は終了後に会場でメモしたものを[ノベルティグッズ管理表](https://docs.google.com/spreadsheets/d/1N73mciqvlQbuC4457OMteS-TXDldj75XybDKfK8CToI/edit?gid=345022493#gid=345022493)の S列「OSC Tokyo Fall 2024開催後在庫」に入れました。
次回はグッズ一覧表を印刷（※）して会場に用意していった方が記録と反映が簡単になるので、そのようにしたいと思います。
※過去列はいらないので必要な列だけ表示するなど多少加工します

色々な方がブースにいらっしゃったのが印象的でした（工業英検を持っている方など）。
以下、当日にぐれさんと会話したことなどのメモです。
・初心者（翻訳することに対して敷居がある人）に対してのアプロ―チがもっと必要
・オフライン開催の場合、環境構築（NodeやGitなど）済の PC を用意すると良いのでは

オフライン開催を復活させたいというのもありつつ、次回（2月）は東京でもセミナーありとのことなので、次回はブース出展だけでなくセミナー開催（ハンズオン形式）もしたいです。春の話は別途ディスカッションを作成します。
